index.html

```diff
@@ -0,0 +1,1547 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
+    crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css" />
+  <title>NewsWeek Clone</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <nav class="d-flex px-4 justify-content-between align-items-center">
+        <div class="d-lg-block d-none d-sm-none d-md-none">
+          <ul class="d-lg-flex  list-unstyled align-items-lg-center">
+            <li>
+              <a href=""><img class="weather-logo" src="./images/sun-cloud.png" alt=""></a>
+            </li>
+            <li class="text-white"> 26°Kampala, UG </li>
+          </ul>
+          <ul class="list-unstyled">
+            <li class="text-white">Fri, Aug 21, 2020</li>
+          </ul>
+        </div>
+        <div>
+          <img class="newsweek-logo ml-3" src="images/newsweek-logo.svg" alt="NewsWeek-logo" />
```

Comment:
- [ ] The header is still not fixed.

| Yours | Original |
|:---:|:---:|
![image](https://user-images.githubusercontent.com/61479648/91076511-040f4500-e616-11ea-93ae-183f2dbbcde4.png) | ![image](https://user-images.githubusercontent.com/61479648/91076729-4afd3a80-e616-11ea-8d43-00b1c6c07912.png) |